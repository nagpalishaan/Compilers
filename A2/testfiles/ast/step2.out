Program
  List
    Function
      IdDecl ID="a"
      List
        DeclareVar
          IdDecl ID="x"
        DeclareVar
          IdDecl ID="y"
        AssignVar
          IdDecl ID="x"
          Numeral NUMERAL="5"
        AssignVar
          IdDecl ID="y"
          IdUse ID="x"
