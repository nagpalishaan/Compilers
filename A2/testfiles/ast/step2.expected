Program
  List
    Function
      IdDecl ID="a"
      Opt
        List
          DeclareVar
            IdDecl ID="x"
            List
          DeclareVar
            IdDecl ID="y"
            List
          AssignVar
            IdDecl ID="x"
            Numeral NUMERAL="5"
          AssignVar
            IdDecl ID="y"
            IdUse ID="x"
